libs/ui/dataviz/src/lib/table/table.component.html

```diff
@@ -32,9 +33,14 @@
         [class.active]="row.id === activeId"
       ></tr>
     </table>
-  </cdk-virtual-scroll-viewport>
-  <div class="text-gray-900 border-t border-gray-300 px-4 py-2 text-sm">
-    <span class="count font-extrabold text-primary">{{ count }}</span
-    >&nbsp;<span translate>table.object.count</span>.
   </div>
+
+  <mat-paginator
+    class="sticky bottom-0 right-0 rounded-lg"
+    (page)="setPagination()"
+    [length]="count"
+    [pageSize]="10"
+    [showFirstLastButtons]="true"
+    [hidePageSize]="true"
+  ></mat-paginator>
```

Comment:
Was there a reason not to use the pagination components already present in gn-ui?

Reply:
Ok, it looks like our pagination components do not provide enough information, that makes sense

Reply:
Yep, the `mat-paginator` felt more appropriate and easier to integrate.